CMakeLists.txt

```diff
@@ -51,6 +51,10 @@ else ()
    message(STATUS "BRAHMA_LIBDIR set to ${BRAHMA_LIBDIR}")
 endif ()
 
+include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules/brahma-utils.cmake)
+install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules/brahma-utils.cmake"
+        DESTINATION "${BRAHMA_LIBDIR}/cmake/brahma")
+
```

Comment:
```suggestion
```
Not needed as install happens [here](https://github.com/hariharan-devarajan/brahma/blob/develop/CMakeLists.txt#L456)

Reply:
This doesn't work as you explained, not sure why. Kept as is for now.